.github/workflows/main.yml

```diff
@@ -34,21 +34,10 @@ jobs:
         minimum-size: 8
         maximum-size: 16
         disk-root: "D:"
-    - name: Setup Java 9
+    - name: Setup Java 17
       uses: actions/setup-java@v1.4.3
       with:
-        java-version: '9'
-        java-package: jdk
-        architecture: x64
-    - name: set JDK_9 environment variable for kotlin compiler
-      shell: bash
-      env:
-        ACTIONS_ALLOW_UNSECURE_COMMANDS: true
-      run: echo ::set-env name=JDK_9::$(echo $JAVA_HOME)
```

Comment:
I'm not sure what the purpose of this was. I don't see any traces of Java 9 anywhere else

Reply:
The test framework depended on JDK_9. I need to check whether that is still the case.

Reply:
I don't see any usage of the `JDK_9` variable on any sources (including the test framework). The tests also pass locally at least.